### PR TITLE
create packForPAW function and tests

### PR DESCRIPTION
**Pull request by flopez-bao**
- create packForPAW function
- remove pop data pull from packForDATIM
- add relevant tests for both functions

### CODE REVIEW

**Comment by flopez-bao** [comment]
Added a `createPAWExport` as a potential route option -- I think it may be the most straightforward after more testing, you can run the COP24 Cameroon datapack to see the differences in the final object, the PAW export should have the extra key pop indicators:

```
library(datapackr)
library(magrittr)

# Point to DATIM login secrets ####
secrets <- Sys.getenv("SECRETS_FOLDER") %>% paste0(., "datim.json")

datimutils::loginToDATIM(secrets)

output_folder <- Sys.getenv("OUTPUT_FOLDER") %>% paste0(., "COP24 Data Packs/")
model_data_path <- Sys.getenv("MODEL_DATA_PATH")
snuxim_model_data_path <- Sys.getenv("SNUXIM_MODEL_DATA_PATH")

# Unpack Submitted Data Pack ####
d <- datapackr::unPackToolSet(file.choose()) 


paw_ex <- createPAWExport(d)
datim_ex <- createDATIMExport(d)
```

**Comment by JordanBalesBAO** [comment]
NOTE for myself, HAVE TO do in an ifelse in the app send to paw button based upon cop year.  This will go away after this year

File: R/createPAWExport.R [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      },
+      error = function(e) {
+        stop("An error occurred for type ", type, ": ", conditionMessage(e))
+      }
+    )
+  }
+
+
+  if (d$info$tool == "Data Pack") {
+    if (d$info$cop_year == 2022) {
```
Comment (JordanBalesBAO) [inline R/createPAWExport.R:27]: Lets dump this 22 if statement.

Comment (JordanBalesBAO) [reply R/createPAWExport.R:27]: Only once we confirm 2022 is no longer supported

**Comment by JordanBalesBAO** [comment]
> NOTE for myself, HAVE TO do in an ifelse in the app send to paw button based upon cop year. This will go away after this year

Won't be necessary the way pack for paw was coded.

**Review by jason-p-pickering — CHANGES_REQUESTED** [review]
I am not really totally comfortable to simply duplicate these two functions. They are essentially 95% the same, so why not just have a common function which handles the common parts, and then handle the specific parts in smaller bits of code?

**Pull request closed without merging**